src/content/it/fundamentals/performance/optimizing-content-efficiency/index.md

```diff
@@ -2,23 +2,32 @@ project_path: /web/_project.yaml
 book_path: /web/fundamentals/_book.yaml
 description: La quantità di dati scaricati da ogni app cresce costantemente. Per garantire prestazioni ottimali dobbiamo ottimizzare l'utilizzo di ogni singolo byte!
 
-{# wf_updated_on: 2014-04-28 #}
+{# wf_updated_on: 2017-11-09 #}
 {# wf_published_on: 2014-03-31 #}
 
 # Ottimizzazione dell'efficienza dei contenuti {: .page-title }
 
 {% include "web/_shared/contributors/ilyagrigorik.html" %}
 
+La nostra applicazione web continua a crescere in termini di ambito di
+applicazione, ambizioni e funzionalità, ed è un bene. Tuttavia, la marcia
+inarrestabile verso un web sempre più ricco porta con sé un'altra tendenza: la
+quantità di dati scaricati da ogni applicazione continua a crescere
+incessantemente. Per garantire prestazioni ottimali dobbiamo ottimizzare
+l'utilizzo di ogni singolo byte di dati!
 
 
-La nostra applicazione web continua a crescere in termini di ambito di applicazione, ambizioni e funzionalità, ed è un bene. Tuttavia, la marcia inarrestabile verso un web sempre più ricco porta con sé un'altra tendenza: la quantità di dati scaricati da ogni applicazione continua a crescere incessantemente. Per garantire prestazioni ottimali dobbiamo ottimizzare l'utilizzo di ogni singolo byte di dati!
+Come appare una moderna applicazione web?
+[HTTP Archive](http://httparchive.org/){: .external } può aiutarci a rispondere
+a questa domanda. Il progetto traccia il modo in cui il web è costruito,
+infiltrandosi periodicamente nei siti più popolari (300.00+ in base all'elenco
```

Comment:
indicizzando periodicamente i siti più popolari (oltre 300.000 in base all'elenco